### PR TITLE
Day16: Functional tests

### PR DESCRIPTION
**Pull request by sergey-bleih**
Doing the most obvious tests (not all the negative or border edge cases). Aiming to cover controllers and repositories, as those are hard to cover with proper unit tests.

### CODE REVIEW

File: .env.test [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# This file is a "template" of which env vars need to be defined for your application
```
Comment (gregurco) [inline .env.test:1]: What do you think about keeping in `.env.test` only the diff drom `.env`? So, if MAILER_URL is different, then we add it to this file. In case of `APP_SECRET`, I think it can be removed from this file and so on.

File: config/bundles.php [hunk 2]
```diff
@@ -21,4 +21,5 @@
     FOS\RestBundle\FOSRestBundle::class => ['all' => true],
     JMS\SerializerBundle\JMSSerializerBundle::class => ['all' => true],
     Symfony\WebpackEncoreBundle\WebpackEncoreBundle::class => ['all' => true],
+    Liip\FunctionalTestBundle\LiipFunctionalTestBundle::class => ['dev' => true, 'test' => true],
```
Comment (gregurco) [inline config/bundles.php:24]: do we need it in dev too?
